lms/urls.py

```diff
@@ -33,6 +33,11 @@
     url(r'', include('student.urls')),
     # TODO: Move lms specific student views out of common code
     url(r'^dashboard$', 'student.views.dashboard', name="dashboard"),
+
+    url(r'^course/{}/refund_status$'.format(settings.COURSE_ID_PATTERN),
```

Comment:
@clintonb I do not fully agree. I think this is related to refund. We may need to refund without enroll or unenroll in future cases. Therefore, I want to separate the money exchange from course enrollment functionality